README.md

```diff
@@ -96,34 +96,62 @@ Install [Python3](https://www.python.org/downloads/) and [VLC](https://www.video
 
 Download Qualcoder-master Zip file and copy it into /Applications
```

Comment:
@ccbogel you might want to fix the above instruction as it falls out of line with what the step-by-step instructions guide to (downloading and extracting the release zip being the last point).

Also, you might want to get rid of the duplicate wording above regarding installing Py & VLC? ;)